mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -267,23 +266,11 @@ def process_api_requests(
             else:
                 next_request = None
 
-            active_run_stack = _active_run_stack.get()
-
-            def call_api(requester, status_tracker, callback_handlers):
-                # langchain inference tracing will read current active run,
-                # but active run stack is thread local, to make it work,
-                # copy current thread active run stack to inference worker thread.
-                _active_run_stack.set(active_run_stack.copy())
-                return requester.call_api(
-                    status_tracker=status_tracker, callback_handlers=callback_handlers
-                )
-
             # if enough capacity available, call API
             if next_request:
                 # call API
                 executor.submit(
-                    call_api,
-                    requester=next_request,
+                    next_request.call_api,
```

Comment:
what is this change for?

Reply:
This part change is introduced in this commit https://github.com/mlflow/mlflow/commit/487db1f6c9b62c40ccfcba76c25defdabbcc4acc

I revert it in this PR, because this part change can't fully address tracing source run lost issue (it only prevents tracing source run lost for MLflow pyfunc model)

Reply:
ah I see, thanks for the explanation!